wiki/Article_Style_Guide/en.md

```diff
@@ -1,192 +1,278 @@
+- Reference style preferred over inline style (reference style is reusable).
+  - Put all the reference links at top of the page.
+  - Reference naming should _make sense_ to the place you will be directed to (like [google homepage link], [Multi_userlog image], or [Sudden Death wikilink]).
+- Use Markdown linkage for page traversal.
+- When translating, keep the **english link reference name**.
+  - This is to ease maintenance in the event of image/link problem (direct copy and paste, rather than hunting for that one link to be fixed).
+
+Markdown:
+```Markdown
+Reference style:
+[page text][my-reference]
+...
+[myreference]: https://www.google.com "Omnipotent searchbar"
+
+Inline style:
+[page text](link "hover-over text")
+```
+
+#### External links
+- `https://` preferred.
+- Clean and direct links only; No third-party shorten links, no links to ads, no google search's complicated link.
+
+#### Wiki Internal links
+- When linking to other pages, you do not need to specify the specific language file (done by osu!web with 'locales').
+  - Capitalisation and punctuation is strict (must be spelt exactly or _404 not found_) when directing to the page.
+    - Linking to osu!catch: `/wiki/Game_Modes/osu!catch/`
+  - For directing to a specific header in the page, the header name must be in lowercase and spaces/punctuations are to be replaced by hyphen (`-`).
+    - Linking to specific section of FAQ: `/wiki/FAQ#ive-never-played-this-ouendan-thing---what-is-it/`
+
+## Images
+**_Image display on the page:_**
+- Image display is as is without any formatting at all.
+  - It is best not to nest images inside GFM's table. (?)
+- For mobile display, hover-over text can be displayed by tapping on the image. The display of the hover-text may show up in extreme-left side of the mobile screen, rendering the alt-text unreadable. If the image was clickable, the hover-over text will not be displayed, and you will be directed to the link instead.
+  - Known to work (hover-over text): Google's Chrome
+  - The rest of the mobile browsers either does not support it, removed by compression (Opera Mini's Extreme Mode), or assumed you meant zoom-in/out (double-tap; depending on browser's settings).
+
+```Markdown
+Add an exclamation mark, '!', in front.
+![broken text](link "Alt text"), or   [broken text]: http://www.google.com "Omnipotent searchbar"
+                                      ...
+                                      ![broken text][my-reference]
+
+Alternatively, clickable image (from @MegaApplePi):
+[![alt-text](link-to-thumb.png) "Click to view full size"](link-to-full.png)
+
+or
+
+[thumb1]: link-to-thumb.png "Click to view full size"
+[full1]: link-to-full.png
+...
+[![alt-text][thumb1]][full1]
+```
+
+**_Image storage:_**
+- Put **images relevant only to the page** at the **same level as the page itself**.
+  - For older images, put in a deprecated folder (?).
```

Comment:
I don't think this is a standard (yet?). Most deprecated images just sit on my PC until I see a fit to use them in HOO.